packages/react-date-time/src/components/DatePicker/DatePickerUtils.ts

```diff
@@ -0,0 +1,52 @@
+export enum DatePickerLocales {
```

Comment:
is this needed? flatpickr exports
https://github.com/flatpickr/flatpickr/blob/f4478d27fce40532298a738aab428b3bbbff751c/src/types/locale.ts#L113

Reply:
ah good call.

Reply:
except, if we import that, there is no beautiful way to document the options. So I figured building an enum that maps to flatpickr type allowed me possibly include the options in the docs and be sure the consumer is using a valid locale code.